lib/objectBuilder.js

```diff
@@ -96,7 +98,21 @@ module.exports = function () {
           throw new Error(util.format('Some arguments to %s(%j) were not specified!', this.type, valid.join(',')));
         }
 
-        return _.pick(this, function (value, key) {
+        // TODO: delete after conclusion of Jaeb study
```

Comment:
a) Please mark the range of lines that should be deleted
b) However, I really don't like this code being in the general-purpose objectBuilder. You've got two apparently mutually-exclusive objects that are being combined into one payload and also allowing the payload itself to have a value. Couldn't/shouldn't this logic just live in the caller function?